sdk/download.go

```diff
@@ -151,3 +153,40 @@ func DownloadSDK(version string) error {
 	tarURL := TarballURL(version)
 	return DownloadSignedFile(tarFile, tarURL)
 }
+
+// UpdateSignedFile will download will replace the local file if the
+// published signature doesn't match the local copy
+func UpdateSignedFile(file, url string) error {
+	sigFile := file + ".sig"
+	sigURL := url + ".sig"
+
+	// get local sig
+	localSig := []byte{}
+	if _, err := os.Stat(sigFile); err == nil {
+		localSig, err = ioutil.ReadFile(sigFile)
+		if err != nil {
+			return err
+		}
+	}
+
+	// overwrite local sig with latest
```

Comment:
I'd be inclined to add a `UpdateFile` which downloads to memory or a temp file and only replaces the original if it changed. Not a big deal but it feels cleaner to not be needlessly changing the inode/mtime.